options_view.cpp

```diff
@@ -1,4 +1,7 @@
 #include "options_view.h"
+#include <cassert>
```

Comment:
This is perfect. Always use angled brackets for standard header files.

Reply:
Monkey see, monkey do is most of the time a good policy when editing code 🙈